charts/timescaledb-multinode/values.yaml

```diff
@@ -41,57 +34,85 @@ env:
+      log_lock_waits: 'on'
+      # These values are set as the default data volume size
+      # is small as well.
+      min_wal_size: 256MB
+      max_wal_size: 512MB
+      temp_file_limit: 1GB
+
+    persistentVolume:
+      enabled: true
+      size: 5G
+      subPath: ""
+      mountPath: "/var/lib/postgresql"
+      annotations: {}
+      accessModes:
+        - ReadWriteOnce
 
-resources: {}
-  # If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+data_node:
+  replicas: 3
+  resources: {}
+    # If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+  postgresql:
+    parameters:
+      max_prepared_transactions: 150
+      # This is rather small, but as this Helm Chart may be used to spin up
+      # 1 access node and 4 data nodes on a single minikube/microk8s this is set
+      # to a conservative value
+      shared_buffers: 300MB
+      work_mem: 16MB
+      timescaledb.passfile: '../.pgpass'
+      # These values are set as the default data volume size
+      # is small as well.
+      min_wal_size: 256MB
+      max_wal_size: 512MB
+      temp_file_limit: 1GB
 
-postgresql:
-  databases:
-    - postgres
-    - example
-  parameters:
-    max_connections: 100
-    max_prepared_transactions: 150
-    # This is rather small, but as this Helm Chart may be used to spin up
-    # 1 access node and 4 data nodes on a single minikube/microk8s this is set
-    # to a conservative value
-    shared_buffers: 300MB
-    work_mem: 16MB
-    timescaledb.passfile: '../.pgpass'
-    log_connections: 'on'
-    log_line_prefix: "%t [%p]: [%c-%l] %u@%d,app=%a [%e] "
-    log_min_duration_statement: '1s'
-    log_statement: ddl
-    log_checkpoints: 'on'
-    log_lock_waits: 'on'
-    # These values are set as the default data volume size
-    # is small as well.
-    min_wal_size: 256MB
-    max_wal_size: 512MB
-    temp_file_limit: 1GB
+    persistentVolume:
+      enabled: true
+      size: 5G
+      ## database data Persistent Volume Storage Class
+      ## If defined, storageClassName: <storageClass>
+      ## If set to "-", storageClassName: "", which disables dynamic provisioning
+      ## If undefined (the default) or set to null, no storageClassName spec is
+      ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+      ##   GKE, AWS & OpenStack)
+      ##
+      # storageClass: "-"
+      subPath: ""
+      mountPath: "/var/lib/postgresql"
+      annotations: {}
+      accessModes:
+        - ReadWriteOnce
 
 # https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 nodeSelector: {}
```

Comment:
nodeSelector/tolerations/affinity should basically also be defined per data/access node